website/docs/r/kubernetes_cluster.html.markdown

```diff
@@ -454,6 +494,68 @@ A `ssh_key` block supports the following:
 
 ---
 
+A `sysctl_config` block supports the following:
+
+* `fs_aio_max_nr` - (Optional) The sysctl setting fs.aio-max-nr. Must be between `65536` and `6553500`. Changing this forces a new resource to be created.
```

Comment:
what does this do?

Reply:
https://www.kernel.org/doc/Documentation/sysctl/fs.txt